components/x-teaser-timeline/stories/content-items.json

```diff
@@ -42,7 +42,8 @@
       "height": 1152
```

Comment:
Surprised to see a diff for snapshots in the x-teaser-timeline directory. Is this necessary?

Reply:
it was breaking the build, the appearance of `scrSet` was causing the snapshot tests to fail